src/api/app/models/statistics/maintenance_statistics_collection.rb

```diff
@@ -34,7 +34,7 @@ def project_created_statistic
     end
 
     def release_request_created_statistic
-      MaintenanceStatistic.new(type: :release_request_created, when: request.created_at)
+      MaintenanceStatistic.new(type: :release_request_created, when: request.created_at, id: request.id)
```

Comment:
hm, can we use the request number for the interface instead an internal database id?